junit-platform-engine/src/main/java/org/junit/platform/engine/support/descriptor/DefaultUriSource.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.engine.support.descriptor;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+import java.net.URI;
+import java.util.Objects;
+
+import org.apiguardian.api.API;
+import org.junit.platform.commons.util.Preconditions;
+import org.junit.platform.commons.util.ToStringBuilder;
+
+/**
+ * Default uri-based test source implementation.
```

Comment:
`Default implementation of {@link UriSource}.`

Reply:
✔️